pkg/ibm/resources_container.go

```diff
@@ -80,6 +82,20 @@ func (resources *ResourcesContainer) AllRegions() []string {
 	return allRegions()
 }
 
+func (resources *ResourcesContainer) ToJSONString() (string, error) {
+	type resourcesJSON datamodel.ResourcesContainerModel
+	toPrint, err := json.MarshalIndent(struct {
+		Version string
```

Comment:
Version should be part of `ResourcesContainerModel`. In other repos we consume the output json, and unmarshal it into a `ResourcesContainerModel`. We may want to refer to the collector's version one day in these repos, so it better be there.

Reply:
I did not do that because it is not a resource.. also this way it will not be supported on aws provider

Reply:
@adisos , what do you think? When the analyzer will consume the collector's model, would it need to know the collector's version?

Reply:
It might, for example if supports up to some version of the collector, though may issue error/warning for some unexpected collector's object regardless of checking the specific version..